### ISSUE TITLE
Screen jumps when editing

### ISSUE THREAD

**Issue by TJH58** [wereturtle/ghostwriter#417]
[large junk file.txt](https://github.com/wereturtle/ghostwriter/files/2870610/large.junk.file.txt)

I have installed Ghostwriter on Manjaro Linux with the Cinnimon desktop. The computer is an HP 830 G5 with 1920x1080 resolution. Ghostwriter was installed with pacman from the Manjaro community. I have Ghostwriter maximized and set to a large font, and I use it in focus mode with a single sentence highlighted. Depending on paragraph size, I have only one to three paragraphs on the screen at a time. The file I am currently working on is about 57,000 words long.

When I arrow up or down, the screen sometimes jumps down a couple of paragraphs and hides the paragraph I am working on. If I arrow back up once, my paragraph comes back into focus. I have not seen any other reports of this. Any ideas?

Note: Change the extension of the attached file to .md to open in Ghostwriter.

Tim

**Comment by wereturtle** [comment]
What's your font size?

**Comment by TJH58** [comment]
Thank you for your reply.

I am using Ghostwriter version 1.7.4. I love your software! However, the jumping started with the last update using pacman. I have noticed it jumping when I use a font 20 or larger. The larger the font the greater the jump, since there is less text on the screen. My monitor is 1920x1080. I typically use a size 48 font and typically have Ghostwriter maximized. I can sit back and not have to squint my eyes to see the words.

Thanks again for looking at the issue.

Tim Hayden

---- Original message ----

**Comment by TJH58** [comment]
Update:

As I periodically do, I ran all pacman updates over the weekend, which consisted of about 750 MB of updated packages. Pacman crashed in the middle of updating and left me with a crippled machine. Thankfully the crash did not take down the operating system completely, but I did lose the network, and pacman and many applications, including Ghostwriter, would not run. It took me four hours to research and correct the problems and complete the updates.

Amazingly, everything is now working, including Ghostwriter. The jumping that Ghostwriter was doing is gone! It now works correctly. All I can say is that there must have been an old or corrupt library that was replaced during the update, because Ghostwriter was not one of the updated packages. Wow! Miracles do happen. . . .
